docs/openapi-issuer.yaml

```diff
@@ -0,0 +1,635 @@
+        page:
+          type: number
+          minimum: 0
+        size:
+          type: object
+          properties:
+            w: { type: number, minimum: 0 }
+            h: { type: number, minimum: 0 }
+
+    SignatureField:
+      type: object
+      properties:
+        attrs:
+          oneOf:
+            - $ref: "#/components/schemas/SignatureFieldAttributes"
+            - $ref: "#/components/schemas/SignatureFieldToBeCreatedAttributes"
+        clause:
+          $ref: "#/components/schemas/Clause"
+      required:
+        - unique_name
+        - clause
+
+    DocumentMetadata:
+      type: object
+      properties:
+        title:
+          type: string
+        signature_fields:
+          type: array
+          items:
+            $ref: "#/components/schemas/SignatureField"
+
+      required:
+        - title
+        - signature_fields
+
+    DocumentMetadataList:
+      type: array
+      items:
+        $ref: "#/components/schemas/DocumentMetadata"
+      minLength: 1
+
+    CreateDossierBody:
+      type: object
+      properties:
+        documents_metadata:
+          $ref: "#/components/schemas/DocumentMetadataList"
+      required:
+        - documents_metadata
+
+    DossierDetailView:
+      type: object
+      properties:
+        id:
+          $ref: "#/components/schemas/Id"
+        documents_metadata:
+          $ref: "#/components/schemas/DocumentMetadataList"
+        created_at:
+          $ref: "#/components/schemas/Timestamp"
+        updated_at:
+          $ref: "#/components/schemas/Timestamp"
+      required:
+        - id
+        - documents_metadata
+        - created_at
+        - updated_at
+
+    Timestamp:
+      type: string
+      format: UTCISODateFromString
+      description: A date-time field in ISO-8601 format and UTC timezone.
+      x-import: "@pagopa/ts-commons/lib/dates"
+      example: "2018-10-13T00:00:00.000Z"
+
+    Id:
+      type: string
+      description: Entity Id
+      format: ulid
+      example: 01ARZ3NDEKTSV4RRFFQ69G5FAV
+
+    FiscalCode:
+      type: string
+      description: User's fiscal code.
+      format: FiscalCode
+      x-import: "@pagopa/ts-commons/lib/strings"
+      example: SPNDNL80R13C555X
+
+    GetSignerByFiscalCodeBody:
+      type: object
+      properties:
+        fiscal_code:
+          $ref: "#/components/schemas/FiscalCode"
+      required:
+        - fiscal_code
+
+    SignerDetailView:
+      type: object
+      properties:
+        id:
+          $ref: "#/components/schemas/Id"
+      required:
+        - id
+
+    RequestSignatureBody:
+      type: object
+      properties:
+        dossier_id:
+          $ref: "#/components/schemas/Id"
+        signer_id:
+          allOf:
+            - $ref: "#/components/schemas/Id"
+            - example: "01BX5ZZKBKACTAV9WEVGEMMVS0"
+        expires_at:
+          $ref: "#/components/schemas/Timestamp"
+      required:
+        - dossier_id
+        - signer_id
```

Comment:
Here we lack the forwarding mode whether by message or qrcode.

Reply:
we leave `send_message: true` ?

Reply:
`notify_user_via_io_message`